content/source/docs/enterprise/private/faq.html.md

```diff
@@ -56,7 +56,9 @@ PTFE has the ability to automatically generate a support bundle including logs a
 
 ## Managing Tool Versions
 
-Terraform Enterprise has a control panel that allows admins to manage the versions of Terraform and Packer and their download locations.
+Terraform Enterprise has a control panel that allows admins to manage the versions of Terraform and Packer and their download locations. 
```

Comment:
Should we delete "and Packer" here, since it's a post-Atlas world?

Reply:
yes - +1 to striking enterprise packer from the docs.

Reply:
It's not quite a post-Atlas world for anyone who can still see this version of the tool management, so I'd like to leave that until it really is. :)